pandas/tests/series/test_io.py

```diff
@@ -142,12 +139,31 @@ def test_to_csv_path_is_none(self):
         csv_str = s.to_csv(path=None)
         assert isinstance(csv_str, str)
 
-    @pytest.mark.parametrize('compression, open_func', [
-        ('gzip', gzip.open),
-        ('bz2', bz2.BZ2File),
-        (pytest.param('xz', lzma.open, marks=td.skip_if_no_lzma)),
+    def decompress_file(self, src_path, compression):
```

Comment:
can u move this function to pandas.util.testing as it’s more generally useful for testing